meshery/meshery
# mesheryctl: mesheryctl set kubeconfig

**Description**


Currently, this PR configures meshery with the kubeconfig based on the options provided by the user. 

This PR fixes #146 

#####  How to test?
* from meshery, download the token() by clicking the "Get Token" item in the dropdown.
* for testing GKE, configure your `kubectl` to point to the cluster.
* check if it's working by running `kubectl get all`,
* build mesheryctl and run :
```shell
./mesheryctl system config gke --token "PATH TO TOKEN"
```

#### Checklist

- [] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [] PR title includes `[<name of area>]:<description>`
- [x] minikube
- [x] GKE
- [x] EKS
- [ ] any more?
